common/buildcraft/transport/block/BlockPipeHolder.java

```diff
@@ -725,4 +725,43 @@ public IBlockState getExtendedState(IBlockState state, IBlockAccess world, Block
     public boolean canRenderInLayer(IBlockState state, BlockRenderLayer layer) {
         return layer == BlockRenderLayer.CUTOUT_MIPPED || layer == BlockRenderLayer.TRANSLUCENT;
     }
+
+    @Override
+    public boolean canConnectRedstone(IBlockState state, IBlockAccess world, BlockPos pos, @Nullable EnumFacing side) {
+        if (side == null)
+            return false;
+        TilePipeHolder tile = getPipe(world, pos, false);
+        if (tile != null) {
+            PipePluggable pluggable = tile.getPluggable(side.getOpposite());
+            return pluggable != null && pluggable.definition == BCTransportPlugs.gate;
```

Comment:
Is there some way to make this not hardcoded? I'd like to support other pluggables or even other types of PipeBehaviour that can connect to redstone.